tests/test_utils.py

```diff
@@ -120,3 +122,25 @@ def test_get_k8s_model():
         'post_start': None,
         'pre_stop': {'exec': {'command': ['/bin/sh', 'test']}},
     }
+
+
+def test_asyncio_atexit():
```

Comment:
Maybe add this to a new test file associated with testing content part of the new file you created? `test_utils.py` with a file named `utils.py` makes me think it only tests things in utils.py.